lib/preprocessors/noop.js

```diff
@@ -0,0 +1,25 @@
+const parsedFiles = new Set();
+
+module.exports = {
+  registerParsedFile(f) {
+    parsedFiles.add(f);
+  },
+  preprocess: null,
+  postprocess: (messages, fileName) => {
+    const msgs = messages.flat();
+    if (
+      msgs.length === 1 &&
+      msgs[0].ruleId === null &&
+      msgs[0].message.startsWith('Parsing error: ')
+    ) {
+      if (!parsedFiles.has(fileName)) {
+        msgs[0].message += '\n';
+        msgs[0].message +=
+          'To lint Gjs/Gts files please follow the setup guide at https://github.com/ember-cli/eslint-plugin-ember#gtsgjs-eslint-setup';
```

Comment:
When is this message going to show up? I do not want to nag all users about a feature they may not care about. Shouldn't it only show up when they have Gjs/Gts files without specifying the override for them?

Reply:
yes, only shows up for gts&gjs files when not correctly setup